app/test_main.py

```diff
@@ -1,3 +1,58 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_ages",
+    [
+        pytest.param(
+            0,
+            0,
+            [0, 0],
+            id="should return [0, 0] when inputs are equal to zeros"
+        ),
+        pytest.param(
+            14,
+            14,
+            [0, 0],
+            id="should return [0, 0] when inputs are "
+               "one less than 1 human year"
+        ),
+        pytest.param(
+            15,
+            15,
+            [1, 1],
+            id="should return [1, 1] when inputs are exactly 1 human year"
+        ),
+        pytest.param(
+            23,
+            23,
+            [1, 1],
+            id="should return [1, 1] when inputs are "
+               "one less than 2 human years"
+        ),
+        pytest.param(
+            24,
+            24,
+            [2, 2],
+            id="should return [2, 2] when inputs are exactly 2 human years"
+        ),
+        pytest.param(
+            27,
+            28,
+            [2, 2],
+            id="should return [2, 2] when inputs are "
+               "one less than 3 human years"
```

Comment:
what is this phrase even supposed to mean?

Reply:
This phrase is supposed to mean that the test covers the case where both inputs are exactly one year short of reaching 3 human years, i.e. the upper boundary of the third equivalence class.
I'll see what I can do to make it easier to understand.